src/seedu/addressbook/parser/Parser.java

```diff
@@ -79,6 +79,9 @@ public Command parseCommand(String userInput) {
         case DeleteCommand.COMMAND_WORD:
             return prepareDelete(arguments);
 
+        case seedu.addressbook.commands.EditCommand.COMMAND_WORD:
```

Comment:
Try to standardize the way the cases are written

Reply:
Yeah idk why this one refused to work until I specified the path. I wanted to just write EditCommand but it kept saying it couldn't find it.